.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "name": "Debug Tiled Server",
+        "type": "python",
+        "request": "launch",
+        "module": "tiled",
+        "args": [
+          "serve",
+          "config",
+          "--host", "0.0.0.0",
```

Comment:
It's generally not best practice for a development container with a placeholder API key (`"secret"`) to accept external connections. Is the default (`127.0.0.1`) not sufficient?

Reply:
Oh yep good catch, 127.0.0.1 should work.

Reply:
Ohh 127.0.0.1 works :D This launch command would (in many cases) be running inside of the dev-container, so I didn't think 127.0.0.1 would allow the mapping of the container port to the host port (8000:8000)

Reply:
Updated

Reply:
If we don't want to map the container port to the host port we could change the docker-compose to use `expose` instead of `ports`

Reply:
I think this should be good. 
Should I remove this change to the compose?
```
    ports:
      - "127.0.0.1:3000:3000"
 ```